### DISCUSSION TITLE
what did @theia/node-pty do?

### DISCUSSION THREAD

**Discussion by linhanyu** [eclipse-theia/theia#9811] [Q&A] [answered]
I have seen vscode and terminus update node-pty to ^0.10.1  to support new version of electron.

When I rebuild native dependency with modified electron version(11.3.0), it's failed in @theia/node-pty because of the binding.gyp of c++ version of this package is c++11 (electron 11 required c++14 support).

Is there any plan to open source @theia/node-pty?

**Comment by vince-fugnitto** [comment, marked as answer]
> Is there any plan to open source @theia/node-pty?

@linhanyu thank you for the discussion, the framework does not use anything closed-source, `@theia/node-pty` is open-source:
- https://github.com/theia-ide/node-pty

As for what it's uses are I believe the `readme` for the repo itself would be the best reference.

**Reply by linhanyu** [reply]
Thanks.I have create a PR to update c++14 support.
